packages/flutter_test/lib/src/binding.dart

```diff
@@ -152,7 +152,8 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   /// environment variables for a variable called `FLUTTER_TEST`.)
   static WidgetsBinding ensureInitialized() {
     if (WidgetsBinding.instance == null) {
-      if (isBrowser || Platform.environment.containsKey('FLUTTER_TEST')) {
+      if (isBrowser || Platform.environment.containsKey('FLUTTER_TEST')
```

Comment:
Why doesn't flutter_test use the same `platform` abstraction that we have in flutter_tools?

Reply:
flutter_tools has a DI system which injects it which we don't want to leak to flutter_test since it involves a ton of dependencies